### PR TITLE
Tiny loadout patch

### PR DESCRIPTION
**Pull request by Timemaster99**
## About the PR
Fixed a couple of loadout issues introduced by #1269 

## Technical details
Changed HoPMess loadout item to HoPMessSkirt, and added the jumpsuit variation
Added minLimit for salvage and chem neck slots, so they aren't forced to wear a neck item
Replaced RD cloak with MG cloak and added the elusive MG beret to loadouts

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
Tiniest changes, so it probably doesn't need a CL.

### CODE REVIEW

File: Resources/Prototypes/Loadouts/Jobs/Command/head_of_personnel.yml [hunk 1]
```diff
@@ -28,13 +28,22 @@
...
     jumpsuit: ClothingUniformJumpskirtHoPMess
+    
+- type: loadout # DeltaV
+  id: HoPMessSuit
+  equipment: HoPMessSuit
+
+- type: startingGear # DeltaV
+  id: HoPMessSuit
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitHoPMess
```
Comment (Axionyxx) [inline Resources/Prototypes/Loadouts/Jobs/Command/head_of_personnel.yml]: These should be added to our own loadouts file

**Review by Axionyxx — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Loadouts/Jobs/Command/head_of_personnel.yml (hunk 1 above)
Comment (Timemaster99) [reply Resources/Prototypes/Loadouts/Jobs/Command/head_of_personnel.yml]: Moved all the items from the loadout PR to our own file now

**Comment by Timemaster99** [comment]
Added MG cloak and beret to this as well

**Review by ghost — APPROVED** [review]
Was about to say "The HoP's winter boots have a spelling error" but they seem to just be like that for the item itself. Regardless of that, this is a quality of life thing that I've been wondering if there was a fix to FOR SOME TIME, so I'm glad to see it.
Null, by the way, dismiss your review on this, the changes were made.

**Review by Axionyxx — APPROVED** [review]

**Pull request merged**